### PR TITLE
Add support for setting sourceRoot

### PR DESCRIPTION
**Pull request by crcn**

### CODE REVIEW

**Comment by michael-ciniawsky** [comment]
@crcn Please rebase, close and reopen the PR to trigger the CLA bot it should be working now 😛 

As a note, I'm in favour of this, but it's not my decision alone so this definitely needs a second pair of eyes before the PR gets merged. Can you also please recap what's the particular usecase ? Thx in advance :)

**Comment by d3viant0ne** [comment]
@crcn - You will need to close and re-open `this` pull request to trigger the CLA assistant so it can be signed.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "css-loader",
-  "version": "0.26.1",
+  "version": "0.26.2",
```
Comment (d3viant0ne) [inline package.json]: Do not bump the version in a pull request, we will handle that on release

**Comment by michael-ciniawsky** [comment]
Thx, but according to Tobias the `sourceRoot` should be empty and loaders shouldn't modify

**Pull request closed without merging**